### DISCUSSION TITLE
Shopify Testify support?

### DISCUSSION THREAD

**Discussion by kenyee** [sergio-sastre/AndroidUiTestingUtils#87] [General]
Curious why you haven't added Testify support as well?

**Comment by sergio-sastre** [comment]
@kenyee
I believe you mean for Cross-Library Screenshot testing? 

If that is the case...
Long answer short:I've been working last 2 weeks on it, and it'll be likely released next week (at least support for Views).

Long answer: 
There were 2 main issues with ndpt/android-testify:
1.  It still uses the deprecated ActivityTestRule, instead of ActivityScenarios.
2. It requires your tests to be annotated.

The first one is not a big deal since one can use TestRules instead of ActivityScenarios, but I found out a bug in android-testify for Composables with Landscape orientation. That's why I'll probably release support for Views first.

The second one requires your current cross-library screenshot tests to add an annotation. The default annotation is that from testify @ScreenshotInstrumentation. But one can define which annotation is used in the gradle plugin to avoid direct dependency to android-testify, e.g. by using your own or one inside AndroidUiTestingUtils.

**Reply by sergio-sastre** [reply]
Just an update.
I’ve made it work with Composables as well 🎉

It‘s very likely I‘ll release a new version supporting Cross-Library Screenshot tests with android-testify next week 😊

**Comment by kenyee** [comment]
Great to hear.
I was wondering whether you didn't support it because Shot was better 🙂

Some of the info you mentioned here might also be good for a blog...i.e., which libraries are harder to use effectively and which have better comparisons.  In this case, it'd be annotations.  Or that Dropshots requires comparison on device so the test APK is larger.
